app/src/main/java/mn/devfest/api/model/Session.java

```diff
@@ -102,6 +113,11 @@ public String getCategory() {
     }
 
     public DateTime getStartTime() {
-        return startTime;
+        try {
+            return DateTime.parse(startTime);
+        } catch (Exception e) {
+            Timber.e(e, "Failed to parse the startTime");
+            return new DateTime();
+        }
```

Comment:
Why keep the start and end times in memory as Strings instead of DateTimes when they are only used as DateTimes? If we call this method multiple times, we incur the overhead of the conversion every time.